lib/gli/commands/help_modules/global_help_format.rb

```diff
@@ -11,7 +11,7 @@ def initialize(app,sorter,wrapper_class)
         end
 
         def format
-          program_desc = @app.program_desc
+          @program_desc = @app.program_desc
```

Comment:
Same issue as above—this makes this code more complex so I'm not comfortable with this change.